content/en/data_security/agent.md

```diff
@@ -88,6 +88,74 @@ If you have a requirement to avoid storing secrets in plaintext in the Agent's c
 
 For more information, see the [Secrets Management][18] documentation.
 
+## Telemetry collection
+
+{{< site-region region="gov" >}}
+
+Datadog on non-government sites will collect environmental, performance, and feature usage information about the Datadog Agent. When Agent detects a government site or FIPS usage Agent will automatically disable this telemetry collection. When such detection is impossible (for example when proxy is used) Agent telemetry will be emitted but immediately dropped at intake. To avoid this emission in the first place we recommend to disable Agent telemetry explicitly by updating the `agent_telemetry` setting in the Agent configuration file, as shown in the example below.
```

Comment:
Would be useful to also mention the corresponding env var as an option to disable telemetry (`DD_AGENT_TELEMETRY_ENABLED=false` I guess)

Reply:
Wow, I had no idea that this would automatically work. I test it and it did. Thank you.

Reply:
done